runtime/parachains/src/runtime_api_impl/vstaging.rs

```diff
@@ -118,3 +118,9 @@ pub fn backing_state<T: initializer::Config>(
 pub fn async_backing_params<T: configuration::Config>() -> AsyncBackingParams {
 	<configuration::Pallet<T>>::config().async_backing_params
 }
+
+/// Implementation of the `minimum_backing_votes` function of the runtime API.
```

Comment:
nit: The comment is a bit superfluous. It should simply say what the function returns.